.github/workflows/publish-to-pypi.yml

```
@@ -64,10 +64,16 @@ jobs:
   Linux:
     strategy:
       matrix:
-        python-version: [ 'cp36-cp36m', 'cp37-cp37m', 'cp38-cp38', 'cp39-cp39' ]
+        py-ver:
```

Maybe just be more explicit here like this?
```suggestion
        python-version:
          - { name: '3.6', manylinux: 'cp36-cp36m' }
          - { name: '3.9', manylinux: 'cp39-cp39m' }
```

---

Indeed.... Done!